### ISSUE TITLE
Get array/string/object length through helper for branching

### ISSUE THREAD

**Issue by dscottboggs** [sunng87/handlebars-rust#428]
For example, I want to do

```handlebars
{{#if len items eq 1}}
Item: {{items.[0]}}
{{ else }}
Items: {{ join items }}
{{/if}}
```
but helpers only support writing to output, not returning a value for comparison. Is there a way to write a helper which assists in branching like this?

**Comment by dscottboggs** [comment]
Never mind, this is resolved by #421. Could this be documented?

**Comment by sunng87** [comment]
I will. Thanks!
